### ISSUE TITLE
Support Season Packs in Series Sorter

### ISSUE THREAD

**Issue by ItsME6969** [sabnzbd/sabnzbd#2171]
Feel free to delete this issue as its a dupe of https://github.com/sabnzbd/sabnzbd/issues/2112
I commented on that issue a little over a month ago and the issue was already closed so maybe my comment wasn't seen but I'll just copy-paste my comment.
```
I thought I was also trippin but it happens quite alot with season packs.
I have sorting as Show Name/Season 1/Show Name - S01E05.mkv
When this error happens, mostly the Show Name also fails to rename properly and gets truncated and what was Show.Name.S01.S01.1080p.AMZN.WEB-DL.DDP5.1.H.264 ends up as Show Name S01 1080P Amzn Web-Dl Ddp5 1 H.
The folder name clearly has S01 but the resulting structure is Show Name S01 1080P Amzn Web-Dl Ddp5 1 H/Season 2/...
Note the Season 2 and how the H.264 lost the .264.
And then the last episode of the pac is the one that mostly fails to rename for me and usually gets labelled as S02E64. Presumably the 2 and 64 came from H.264 and SAB took that as the season and episode numbers. So it ends up as
Show Name S01 1080P Amzn Web-Dl Ddp5 1 H - S02E64.mkv without the episode number and I have to rename that file with the last episode number.
The same nzb with nzbget and post processing enable works with no problem.
```

**Comment by Safihre** [comment]
Can you share an example NZB? Here or at safihre@sabnzbd.org
And your exact Sorting string.

**Comment by ItsME6969** [comment]
I just sent you a random NZB that I actually just tried right now before sending. I dont do alot of season packs but all the ones that I get have the same error even this one that I just randomly from the indexer so presumably it bugs out on all season packs. Single episodes work perfectly tho.

I'm on SAB 3.2.1
`%sn/Season %s/%sn - S%0sE%0e.%ext`

**Comment by Safihre** [comment]
Wait, you are on 3.2.1? Why?

**Comment by ItsME6969** [comment]
😱 lmao its what comes with the box man. Anyway the bug still persists in 3.5.3 which I have locally installed and I just tested and confirmed that with a different season pack. Same bug.

**Comment by Safihre** [comment]
Oke, so I downloaded your example NZB and I get the result I expected.
As far as I know we don't support season-packs and never have, right @jcfp?
So it only renames the first one, and that's it.
![image](https://user-images.githubusercontent.com/5703454/166689069-770b96eb-cfa8-4cee-9dea-5cdc310fbde9.png)

I think part of your original bug is not present anymore in the new version of SABnzbd, since we now use a new parser that correctly parses the `Show Name`.

**Comment by ItsME6969** [comment]
Yes part of the original bug is fixed.
`So it only renames the first one, and that's it.`
But as you see it lost the episode number and renamed episode 8...not the first one.

If season packs are not supported then maybe we can turn this thread to a suggestion. And thanks for taking the time to look into it.

**Comment by Safihre** [comment]
Indeed, I think this is a valid feature request to support season packs. Since we also support multi volume for Movies.
I will rename the issue to reflect that.
Probably best to do it together with #2143, since it needs a good refractor!

**Comment by Safihre** [comment]
Implemented in #2461
